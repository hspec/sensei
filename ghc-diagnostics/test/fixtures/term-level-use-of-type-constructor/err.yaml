version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/term-level-use-of-type-constructor/Foo.hs
  start:
    line: 3
    column: 7
  end:
    line: 3
    column: 10
severity: Error
code: 1928
message:
- Illegal term-level use of the type constructor `Foo'
- '  defined at test/fixtures/term-level-use-of-type-constructor/Foo.hs:2:1'
- |-
  Perhaps use one of these:
    variable `foo' (line 3), `Fooa' (line 2), `Fooi' (line 2)
- |-
  In the expression: Foo
  In an equation for `foo': foo = Foo
hints: []
