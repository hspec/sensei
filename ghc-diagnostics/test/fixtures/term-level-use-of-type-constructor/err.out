test/fixtures/term-level-use-of-type-constructor/Foo.hs:3:7: error: [GHC-01928]
    • Illegal term-level use of the type constructor `Foo'
    • defined at test/fixtures/term-level-use-of-type-constructor/Foo.hs:2:1
    • Perhaps use one of these:
        variable `foo' (line 3), `Fooa' (line 2), `Fooi' (line 2)
    • In the expression: Foo
      In an equation for `foo': foo = Foo

